spec/checkout_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'checkout'
+
+describe Checkout do
+
+  context 'Check out an order' do
+
+    let(:order) { double :order, dishes: [{'Korma': 4.95}, {'Red Curry': 5.95}, {'Red Curry': 5.95}, {'Green Curry': 6.95}] }
```

Comment:
Line is too long. [125/80]<br>Space inside { missing.<br>Space inside } missing.